oar/core/worksheet.py

```diff
@@ -94,7 +94,8 @@ def create_test_report(self):
             # update advisory info
             ad_cell_value = ""
             for k, v in self._cs.get_advisories().items():
-                ad_cell_value += f"{k}: {util.get_advisory_link(v)}\n"
+                ad_link = {util.get_advisory_link(v)}
+                ad_cell_value += f"{k}: {self._report._to_hyperlink(ad_link, ad_link)}\n"
```

Comment:
Multiple `=HYPERLINK` in the same cell does not work as expected.

Reply:
You will need to use a different solution. See suggestions in https://github.com/openshift/release-tests/pull/525#discussion_r2108319973.
The same applies also to master branch. See https://github.com/openshift/release-tests/pull/524#discussion_r2114068697